### PR TITLE
Updating schedule for day 2

### PR DESCRIPTION
**Pull request by eglerean**
changes on schedule for day 2.

### CODE REVIEW

**Comment by rkdarst** [comment]
Modules is also in day 3 (while it doesn't make sense from the flow, it was part of the "distribute the boring parts between the days" idea). If not, you can copy the day3 info on modules stuff to day2.

  Does this make more time for serial jobs on day 1.

**Comment by rkdarst** [comment]
I don't know where these went, but somehow they are missing.  Both wore done first thing on day 2.  They take the place of the day's intro, and I think we could fit in in there with "connecting" which shouldn't take that long - I hope...

https://scicomp.aalto.fi/triton/tut/about-tutorials/
https://scicomp.aalto.fi/triton/tut/intro/

**Comment by eglerean** [comment]
intro and about were somewhat done today, so we can show the pages tomorrow but not spend 20 minutes like we did in the winter as it is mostly repeating what they have heard already

**Comment by rkdarst** [comment]
+1, should be fast

File: training/scip/summer-kickstart.rst [hunk 1]
```diff
@@ -192,21 +192,36 @@ resources.  All times are EEST (Helsinki) time.
...
       * Aalto: (same)
       * Helsinki: `general information <https://wiki.helsinki.fi/display/it4sci/HPC+SUMMER+KICKSTART+2021>`__
       * Tampere: `Narvi storage <https://narvi-docs.readthedocs.io/narvi/tut/storage.html>`__
       * This topic is *very* site-specific.  The general principles
 	will apply everywhere, but the exact paths/servers will vary.
 
-    * :doc:`/triton/tut/interactive`
+    * 13:50: Break
+
+    * 14:00: Short talk: `Radovan Bast <https://bast.fr/> (UiT The Arctic University of Norway)`__: `Asking for help with supercomputers <https://cicero.xyz/v3/remark/0.14.0/github.com/bast/help-with-supercomputers/main/talk.md/#1>`__
```
Comment (rkdarst) [inline training/scip/summer-kickstart.rst:215]:
```suggestion
    * 14:00: Short talk: `Radovan Bast <https://bast.fr/>`__ (UiT The Arctic University of Norway): `Asking for help with supercomputers <https://cicero.xyz/v3/remark/0.14.0/github.com/bast/help-with-supercomputers/main/talk.md/#1>`__ 
```

**Pull request closed without merging**